dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceAPIFactory.java

```diff
@@ -1,53 +1,14 @@
 package com.dotcms.cms.login;
 
-import com.dotcms.api.web.HttpServletRequestThreadLocal;
-import com.dotcms.auth.providers.jwt.JsonWebTokenUtils;
-import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
-import com.dotcms.util.ReflectionUtils;
-import com.dotcms.util.security.EncryptorFactory;
-import com.dotmarketing.beans.Host;
-import com.dotmarketing.business.APILocator;
-import com.dotmarketing.business.ApiProvider;
-import com.dotmarketing.business.UserAPI;
-import com.dotmarketing.business.web.UserWebAPI;
-import com.dotmarketing.business.web.WebAPILocator;
-import com.dotmarketing.cms.factories.PublicEncryptionFactory;
-import com.dotmarketing.cms.login.factories.LoginFactory;
-import com.dotmarketing.cms.login.struts.LoginForm;
-import com.dotmarketing.exception.DotDataException;
-import com.dotmarketing.exception.DotSecurityException;
-import com.dotmarketing.factories.PreviewFactory;
-import com.dotmarketing.util.*;
-import com.liferay.portal.NoSuchUserException;
-import com.liferay.portal.PortalException;
-import com.liferay.portal.SystemException;
-import com.liferay.portal.auth.AuthException;
-import com.liferay.portal.auth.Authenticator;
-import com.liferay.portal.auth.PrincipalFinder;
-import com.liferay.portal.ejb.UserLocalManagerUtil;
-import com.liferay.portal.ejb.UserManagerUtil;
-import com.liferay.portal.events.EventsProcessor;
-import com.liferay.portal.language.LanguageUtil;
-import com.liferay.portal.model.Company;
-import com.liferay.portal.model.User;
-import com.liferay.portal.servlet.PortletSessionPool;
-import com.liferay.portal.util.PortalUtil;
-import com.liferay.portal.util.PropsUtil;
-import com.liferay.portal.util.WebKeys;
-import com.liferay.util.InstancePool;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.http.HttpSession;
 import java.io.Serializable;
-import java.util.Iterator;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Optional;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)